### PR TITLE
fix: header links UX optimizing

### PR DESCRIPTION
**Pull request by lamartire**
@ifiri ready for review

### CODE REVIEW

File: components/PageHeader.vue [hunk 1]
```diff
@@ -13,21 +13,69 @@
...
           </li>
         </nav>
         <ul class="page-header-menu-list">
           <li>
-            <a href="https://help.endpass.com/" target="_blank">Support</a>
+            <a
+              class="page-header-link"
+              href="https://help.endpass.com/"
+              target="_blank"
+              >Support</a
```
Comment (ifiri) [inline components/PageHeader.vue]: Try to fix this layout please, even if this is a prettier, looks strange

**Comment by dudiq** [comment]
approved 
@kazurus

File: components/PageHeader.vue [hunk 2]
```diff
@@ -13,21 +13,69 @@
           <v-logo></v-logo>
         </div>
         <nav class="page-header-menu-list">
-          <li><nuxt-link to="/">Home</nuxt-link></li>
-          <li><nuxt-link to="/pricing">Pricing</nuxt-link></li>
-          <li><nuxt-link to="/about">About</nuxt-link></li>
-          <li><nuxt-link to="/careers">Careers</nuxt-link></li>
-          <li><nuxt-link to="/blog">Blog</nuxt-link></li>
+          <li v-if="!isHomePage">
```
Comment (kazurus) [inline components/PageHeader.vue]: Why do we need it? Is active class enough?

Comment (lamartire) [reply components/PageHeader.vue]: Because we should not display this link on home page

Comment (dudiq) [reply components/PageHeader.vue]: maybe just make `home` link as opened state here?
i think that it's not good, when top menu items list are changed for different pages

Comment (kazurus) [reply components/PageHeader.vue]: we have underscore for links, and will be good to have consistent menu

**Comment by lamartire** [comment]
@dudiq @kazurus done. But for active links we have bold weight, not underline

**Comment by kazurus** [comment]
@lamartire we should fix design:
image

**Comment by lamartire** [comment]
@kazurus done

**Comment by kazurus** [comment]
@dudiq need you approve again

**Comment by dudiq** [comment]
@lamartire something wrong with underline, when open price page
image

**Comment by lamartire** [comment]
@dudiq again ready

**Comment by dudiq** [comment]
@kazurus approved

**Pull request merged**